amazon-ses-connector-test/src/test/resources/docker-compose-test.yaml

```diff
@@ -0,0 +1,111 @@
+version: '3.9'
+
+x-sso-variables: &sso-variables
+  SIG_SECRET_KEY: JWTSecretKeyDontUseInProduction!
+  EXTERNAL_SIG_SECRET_KEY: JWTSecretKeyDontUseInProduction!
+  EXTERNAL_ENC_SECRET_KEY: JWTEncKeyDontUseInProduction666!
+  sso.jwt.internal.signature.key.type: ENV
+  sso.jwt.internal.signature.key.value: SIG_SECRET_KEY
+
+x-eureka-variables: &eureka-variables
+  eureka.client.service-url.defaultZone: http://registry:8080/eureka
+
+x-actuator-variables: &actuator-variables
+  management.endpoints.web.exposure.include: "*"
+  management.endpoints.enabled-by-default: "true"
+
+x-activemq-variables: &activemq-variables
+  spring.activemq.broker-url: tcp://message-broker:61616
+  spring.activemq.user: admin
+  spring.activemq.password: admin
+
+services:
+
+  registry:
```

Comment:
Do we really need want to include the registry as part of the system tests? this will just make things slower. We rely on a message broker and do not even have a REST route in the service?